js_modules/dagit/packages/core/src/assets/AssetMetadata.tsx

```diff
@@ -13,7 +13,9 @@ export const metadataForAssetNode = (
   assetNode: AssetNodeOpMetadataFragment,
 ): {assetType?: DagsterTypeFragment; assetMetadata: MetadataEntryFragment[]} => {
   const assetType = assetNode.type ? assetNode.type : undefined;
-  const assetMetadata = assetNode.metadataEntries || [];
+  const assetMetadata = (assetNode.metadataEntries || []).filter(
```

Comment:
It looks like there are a few references to `__code_origin` - maybe we pull that out and put it in CodeLink.tsx or CodeLinkProtocol.tsx? (neither of those locations is perfect but it seems like it'd be nice to see based on the file what feature it's linked to.